src/app/modules/main/chat_section/sub_item.nim

```diff
@@ -8,10 +8,10 @@ type
     parentId: string
 
 proc initSubItem*(id, parentId, name, icon: string, isIdenticon: bool, color, description: string, `type`: int, 
-  amIChatAdmin: bool, hasUnreadMessages: bool, notificationsCount: int, muted, active: bool, position: int): SubItem =
+  amIChatAdmin: bool, hasUnreadMessages: bool, notificationsCount: int, muted, blocked, active: bool, position: int): SubItem =
```

Comment:
Without knowing nothing about nim.. should it be defined, as well, that `blocked` is a `bool`? Maybe the same happens with `muted`..

Reply:
@noeliaSD in `Nim` defining proc like `proc procName(param1, param2, param3: AnyNimType)` is the same as `proc procName(param1: AnyNimType, param2: AnyNimType, param3: AnyNimType)`, that's the matter of personal preference or coding style.

Reply:
> @noeliaSD in `Nim` defining proc like `proc procName(param1, param2, param3: AnyNimType)` is the same as `proc procName(param1: AnyNimType, param2: AnyNimType, param3: AnyNimType)`, that's the matter of personal preference or coding style.

Ok! Perfect!! Thanks for the explanaition!